client/src/pages/Orders/Orders.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { CustomButton } from '../../components/common/CustomButton/CustomButton';
+import { ModalWindow } from '../../components/common/ModalWindow/ModalWindow';
+import { OrderCard } from '../../components/OrderCard/OrderCard';
+import { useAppDispatch, useAppSelector } from '../../hooks/redux';
+import { getCurrentUserOrders } from '../../stateManager/actionCreators/orders';
+import { setCancelError } from '../../stateManager/slices/ordersSlice';
+import styles from './Orders.module.scss';
+
+interface IProps {}
+
+export const Orders: React.FC<IProps> = ({}) => {
+    const dispatch = useAppDispatch();
+    const orders = useAppSelector((state) => state.orders.orders);
+    const [isModalActive, setIsModalActive] = useState(false);
+    const cancelError = useAppSelector((state) => state.orders.cancelError);
+
+    useEffect(() => {
+        dispatch(getCurrentUserOrders());
+    }, []);
+
+    useEffect(() => {
+        if (cancelError) {
+            setIsModalActive(true);
+        }
+    }, [cancelError]);
+
+    const renderColumnNames = (text: string) => {
```

Comment:
Please use single form, not plural, because this method only renders one column name